documentation/_Sidebar.asciidoc

```diff
@@ -3,7 +3,7 @@
 * link:Guide-to-the-Reader[Guide to the Reader]
 * link:cobigen-usecases[General CobiGen Use Cases]
 
-==== HowTo's
+==== `HowTo's`
```

Comment:
Backticks in headlines does not work. It would even break docs generation as far as I know. Also HowTo is not a code snippet at all. It's a valuable vocabulary. Please add to the dict.